src/plugins/audioPlugin.tsx

```diff
@@ -102,6 +104,8 @@ export default function createAudioPlugin(options: TransformOptions = {}): Audio
         copyright: audio.copyright,
         src: audio.audioFile.url,
         copyText: copyString,
+        description: audio.podcastMeta?.introduction,
+        resourceOverride: audio.podcastMeta !== undefined ? 'podcasts' : 'audios',
```

Comment:
Kan denne forenkles til dette?:
```suggestion
        resourceOverride: audio.podcastMeta && 'podcasts',
```